### ISSUE TITLE
Auto Disconnect does not work with GCODE

### ISSUE THREAD

**Issue by derpicknicker1** [jneilliii/OctoPrint-Tasmota#58]
My printer will be turned off by GCODE. In this case, it will not get disconnected automatically, even when I activate this functionality.

**Comment by jneilliii** [comment]
It should if you set your delays correctly. Share a screenshot of your settings and I'll tell you what needs to be adjusted.

**Comment by derpicknicker1** [comment]
![image](https://user-images.githubusercontent.com/5983533/74846765-57d06700-5331-11ea-96b6-749fc4763d6a.png)
This is what I have at the moment. I tried several different settings without luck.

**Comment by jneilliii** [comment]
Ok, I went back and looked and you're right.  The current released version has a bug where some of these options don't get triggered properly from gcode. The next upcoming release should resolve this as the logic was moved from the client-side web interface to the server-side.  It also incorporates a new automatic power off feature that gets triggered by the print complete event with an abort option.
